flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/stream/sql/TableSinkITCase.scala

```diff
@@ -399,6 +399,47 @@ class TableSinkITCase(mode: StateBackendMode) extends StreamingWithStateTestBase
     assertThat(actualUseStatement.sorted).isEqualTo(expected.sorted)
   }
 
+  @TestTemplate
+  def testCreateTableAsSelectWithPrimaryKey(): Unit = {
```

Comment:
not sure if this test is really useful. the primary key is not used in this test. even if it were lost in translation, the test would not even notice it. a non-IT case should be enough that accesses the catalog table and this is done by `SqlDdlToOperationConverterTest` already. so let's drop this one.

Reply:
Removed.